docs/INSTALL.md

```diff
@@ -31,7 +31,7 @@
 **Setup manually as follows**
 
 1. Add ansible modules to ANSIBLE_LIBRARY environment variable
-<br/>` echo "export ANSIBLE_LIBRARY=$ANSIBLE_LIBRARY:'/path/to/gdeploy/modules/'" >> ~/.bashrc`<br/>
+<br/>` echo "export ANSIBLE_LIBRARY=$ANSIBLE_LIBRARY'/path/to/gdeploy/modules/'" >> ~/.bashrc`<br/>
```

Comment:
is this correct? As per docs, "$ANSIBLE_LIBRARY takes a colon-separated list like $PATH"  @pkesavap